echopype/convert/set_groups_azfp.py

```diff
@@ -37,6 +37,51 @@ class SetGroupsAZFP(SetGroupsBase):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        # get frequency values
+        freq_old = list(self.parser_obj.unpacked_data["frequency"])
+
+        # sort the frequencies in ascending order
+        freq_new = freq_old[:]
+        freq_new.sort(reverse=False)
+
+        # obtain sorted frequency indices
+        self.freq_ind_sorted = [freq_new.index(ch) for ch in freq_old]
+
+        # obtain sorted frequencies
+        self.freq = self.parser_obj.unpacked_data["frequency"][self.freq_ind_sorted]
+
+        # obtain channel_ids
+        self.channel_ids = self._create_unique_channel_name()
+
+        # Put Frequency in Hz (this should be done after create_unique_channel_name)
+        self.freq = self.freq * 1000  # Frequency in Hz
```

Comment:
is the comment still accurate? but this is in `__init__`?

Reply:
If you are referring to the portion "(this should be done after create_unique_channel_name)", then yes. If we did this before `create_unique_channel_name`, then the name would be say `'55030-125000-1'` instead of `'55030-125-1'`. If you prefer the former, then maybe we should create an issue for this.